src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -322,20 +322,6 @@ def _sanitize_parameters(
         if decoder_kwargs is not None:
             postprocess_params["decoder_kwargs"] = decoder_kwargs
         if return_timestamps is not None:
-            if self.type == "seq2seq" and return_timestamps:
```

Comment:
Can't perform these checks here -> `_santize_parameters` is called before `self.type` is set